bouncer/templates/annotation.html.jinja2

```diff
@@ -1,17 +1,17 @@
 {% extends "templates/base.html.jinja2" %}
 
-{% set title = _("Hypothesis: Redirecting you&hellip;") %}
+{% set title = _("Hypothesis: Annotation for " ) + pretty_url  %}
 
 {% block content %}
   <div class="center spinner__stationary-ring">
-    <img alt="Redirecting"
+    <img alt="Loading annotation for {{ pretty_url }}"
```

Comment:
I see that we weren't doing it before, but this one should be sent through `_()` as well